iothub/device/src/InternalClient.cs

```diff
@@ -1790,12 +1765,42 @@ internal void ValidateModuleTransportHandler(string apiName)
 
         #endregion Module Specific API
 
+        public void Dispose()
```

Comment:
publics above privates

Reply:
does this mean the private methods in the regions above?